include/odp/api/spec/schedule_types.h

```diff
@@ -558,8 +686,24 @@ typedef struct odp_schedule_config_t {
  * Schedule group information
  */
 typedef struct odp_schedule_group_info_t {
-	const char    *name;   /**< Schedule group name */
-	odp_thrmask_t thrmask; /**< Thread mask of the schedule group */
+	/** Schedule group name */
+	const char *name;
+
+	/** Thread mask of the schedule group */
+	odp_thrmask_t thrmask;
+
+	/** Group specific default priority */
+	odp_schedule_prio_t def_prio;
```

Comment:
What is group specific default priority? What signifigance is has in API? Reading the API I cannot really understand it, so some API text improvement is probably needed somewhere. The global default priority returned by odp_schedule_default_prio() is the default value written to odp_schedule_param_t:: prio by odp_queue_param_init(), but there does not seem to be any connection between the group specific priority and the rest of the API.